### ISSUE TITLE
Validation was unable to complete successfully due to an unexpected error.

### ISSUE THREAD

**Issue by machschnell** [mozilla/addons-linter#4067]
I tried to submit an add-on at the Firefox Developer Hub. It fails with "Error: Validation was unable to complete successfully due to an unexpected error." under General Tests without any additional information, I'm reporting it here as requested.

My add-on is attached. It loads fine as Temporary Add-on Firefox 94.0.1. 

I installed addons-linter (version 4.4.0) using npm. I see no errors when I run:

addons-linter "C:\Users\matt\Documents\Eclipse\BBO Helper\Releases\BBO Helper 1.0.zip"

(There are 43 UNSAFE_VAR_ASSIGNMENT warnings, involved innerHTML  insertAdjacentHTML, etc that I know are safe in practice.
[BBO Helper 1.0.zip](https://github.com/mozilla/addons-linter/files/7598900/BBO.Helper.1.0.zip)
)

**Comment by willdurand** [comment]
@machschnell hey, can you give the output of `addons-linter --version`? I am thinking that you are using v4.4.0, which has been released a couple hours ago and that isn't yet what AMO is using (AMO currently uses v4.2.0).

**Comment by machschnell** [comment]
Yes, it was 4.4.0 that I installed locally via npm. The AMO upload interface doesn't indicate which version it is currently using so I just grabbed the latest version via npm.

**Comment by willdurand** [comment]
Ok, so I think this problem will be solved after today's push (i.e. Nov. 25 at 9AM PT).

**Comment by willdurand** [comment]
I am going to close this issue now because it looks very similar to #4068. Let me know if you're still not able to submit your add-on later today.
